Rakefile

```diff
@@ -1,17 +1,26 @@
 require 'rubygems'
 require 'bundler'
+require 'rbconfig'
+# rubocop:disable Style/DoubleNegation
+is_windows = !!(RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/)
```

Comment:
I'm aware this was copied from elsewhere, but a better one would be to use the .match? method

Reply:
I think I'll take a look on this in another PR
That's why I've opened the one to update rubocop. With a more recent version of rubocop, we won't need this anymore

Reply:
Actually, I've just activated rubocop also on windows build, and it is working fine